src/storer.ts

```diff
@@ -4,16 +4,85 @@ export class Storer {
     static azure = require('azure');
     static crypto = require('crypto');
     static config = require('../config.json');
+    static path = require('path');
+    //This could be left out but it makes it easier for the other groups to access the blobs if we store them with fqdn
+    static blobPrefix = 'https://wetstorage.blob.core.windows.net/websites/';
 
     static blobService = Storer.azure.createBlobService(Storer.config.storageAccountname,
         Storer.config.storageKey);
 
     static container = Storer.config.container;
 
-    public static storeWebsite(webpage : WebPage) : void {
-        Storer.storeWebsiteBlob(webpage);
+    private context;
 
-        //TODO: Create database entry for website
+    constructor(){
+        let me = this;
```

Comment:
You don't have to use this work around in ECMA6. Use a lambda function `() => {}` for the callback. Lambda functions in ECMA6 preserve the `this` context.